book/SUMMARY.md

```diff
@@ -104,7 +104,8 @@
 - [Push Plugins](/push/push-general.md)
   - [iOS](/push/push-ios.md)
   - [Android](/push/push-android.md)
-
+- [Crashlog Plugins](/crashlog/crashlog-general.md)
+  - [iOS](/crashlog/crashlog-ios.md)
```

Comment:
@alexzchut isn't it called `Error Monitoring` on zapp? I think it is better to have alignment in names.